@ecomplus/storefront-template/template/js/lib/menu.js

```diff
@@ -1,9 +1,56 @@
-import { animateCss, $ } from '@ecomplus/storefront-twbs'
+import { isMobile, animateCss, $ } from '@ecomplus/storefront-twbs'
 import overlay from './overlay'
 
 const $menu = $('#menu')[0]
 let isVisible = false
 
+let $openSubMenu
+
+const closeSubmenu = () => {
+  $openSubMenu.style.display = 'none'
+  $openSubMenu = null
+}
+
+const toggleSubmenu = (slug, isClick) => {
+  if (isClick && slug) {
+    window.location = `/${slug}`
+    return
+  }
+  if (!isMobile) {
+    if ($openSubMenu || isClick) {
+      closeSubmenu()
+    }
+    if (slug) {
+      $openSubMenu = document.getElementById(`s-${slug.replace(/\//g, '_')}`)
+      if (!$openSubMenu) {
+        return
+      }
+
+      if ($openSubMenu) {
+        $openSubMenu.style.display = 'flex'
+        animateCss($openSubMenu, 'fadeIn')
+        setTimeout(() => { document.addEventListener('click', isOutClicked) }, 200)
+      }
+    }
+  }
+}
+
+const isOutClicked = (e) => {
```

Comment:
`is*` é para booleanos, aqui é uma função, o ideal é o nome ser um verbo, eg.: `checkSubmenuOutClick`, também sugeri um nome menos genérico porque nesse arquivo não temos só handlers do submenu...